src/main/java/controller/GetMessage.java

```diff
@@ -109,7 +114,14 @@ public void getMessageWithoutChatIdPathVariable(HttpServletRequest request) {
         }
 
         Message message = queryMessage.getMessage(messageId);
-    
-        return SuccessResponseGenerator.getSuccessResponseForGetMessage(message, userId);
+        Attachment attachment = null;
+
+        if (message.getAttachmentId().isPresent()) {
+            attachment = queryAttachment.getAttachment(message.getAttachmentId().get());
+
+            return SuccessResponseGenerator.getSuccessResponseForGetMessage(message, attachment, userId);
+        } else {
```

Comment:
You can omit the `else` keyword since the `if` block returns.

Reply:
Done